kura/org.eclipse.kura.wire.provider/src/main/java/org/eclipse/kura/internal/wire/WireServiceImpl.java

```diff
@@ -265,7 +268,7 @@ public void deleteWireConfiguration(final WireConfiguration wireConfiguration) {
                 }
             }
         } catch (final InvalidSyntaxException e) {
-            throw new KuraRuntimeException(KuraErrorCode.INTERNAL_ERROR, ThrowableUtil.stackTraceAsString(e));
+            throw new IllegalArgumentException(ThrowableUtil.stackTraceAsString(e));
```

Comment:
True. But passing in the original exception as a string is a bit against the idea of that. Normally in Java you would pass in the original exception and let Java handle the rest.